src/shared/components/oncoprint/OncoprintUtils.ts

```diff
@@ -193,21 +193,32 @@ export function getHeatmapTrackRuleSetParams(
             value_range = [0, 1];
             legend_label = trackSpec.legendLabel || 'Methylation Heatmap';
             value_stop_points = [0, 0.35, 1];
-            colors = [[0, 0, 255, 1], [255, 255, 255, 1], [255, 0, 0, 1]];
```

Comment:
https://prettier.io/blog/2019/11/09/1.19.0.html#break-arrays-of-arraysobjects-if-each-element-has-more-than-one-elementproperty-6694-by-sosukesuzuki